docs/writing-views.md

```diff
@@ -0,0 +1,301 @@
+# Writing views in Zulip
+
+## What this covers
+
+This page documents how views work in Zulip. You may want to read the 
+[new feature tutorial](https://zulip.readthedocs.io/en/latest/new-feature-tutorial.html)
+or the [integration guide](https://zulip.readthedocs.io/en/latest/integration-guide.html),
+and treat this as a reference.
+
+If you have experience with Django, much of this will be familiar, but
+you may want to read about how REST requests are dispatched, and how
+request authentication works.
+
+This document supplements the [new feature tutorial](https://zulip.readthedocs.io/en/latest/new-feature-tutorial.html)
+and the [testing](https://zulip.readthedocs.io/en/latest/testing.html)
+documentation.
+
+## What is a view?
+
+A view in Zulip is everything that helps implement a server endpoint.
```

Comment:
Am I right that we're using `view` in the classing "Model, View, Controller" sense? If so, it might help to say that.